ovirt-web-ui.spec.in

```diff
@@ -20,19 +20,22 @@ Source0:        https://github.com/oVirt/ovirt-web-ui/archive/%{source_basename}
 
 ExclusiveArch:  x86_64
 
-BuildRequires: ovirt-engine-nodejs = 6.9.1
-# Keep ovirt-engine-nodejs-modules at particular version unless tested on higher
-BuildRequires: ovirt-engine-nodejs-modules = 0.0.17
+# Keep ovirt-engine-{nodejs|nodejs-modules|yarn} at particular version unless tested on higher
+# Please keep in sync with the automation/*.req files
+BuildRequires: ovirt-engine-nodejs=6.9.4
+BuildRequires: ovirt-engine-nodejs-modules=1.0.3
+BuildRequires: ovirt-engine-yarn=0.19.1
```

Comment:
I'm afraid you are mistaken here. The repos are available to the automation scripts, they are just not in the `yum.repos.d` directory. Instead, they are injected directly to the `/etc/yum/yum.conf` (or `/etc/dnf/dnf.conf` where applicable) file.

We actually disable the functionality of the `yum.repos.d` directory inside the mock environment to prevent the installation of "release" RPMs from causing things running in mock from downloading packages from the internet as opposed to the CI system's mirrors and proxies.

The technique of using `yum-builddep` is actually already used successfully by other projects (e.g. engine)